rest-social-media/ballerina/social_media_service_desc.bal

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2023, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/constraint;
+import ballerina/http;
+import ballerina/sql;
+import ballerina/time;
+
+type SocialMedia service object {
+    *http:Service;
```

Comment:
This should be `http:InterceptableService`

```suggestion
    *http:InterceptableService;
```

Reply:
Fixed with [d5a67e32d8f6e96f97721bb0c97cf79234e3c39e](https://github.com/ballerina-guides/integration-samples/pull/29/commits/d5a67e32d8f6e96f97721bb0c97cf79234e3c39e)